code/modules/hydroponics/hydro_tools.dm

```diff
@@ -105,110 +106,105 @@
-	if(grown_seed.highkpa_tolerance > 220)
-		dat += "<br>It is well adapted to high pressure levels."
+		dat += "It does not require any fluids to subsist.<br>"
+	else if(grown_seed.toxin_affinity < 5)
+		dat += "It requires [grown_seed.fluid_consumption] units of water per cycle.<br>"
+	else if(grown_seed.toxin_affinity > 7)
+		dat += "It requires [grown_seed.fluid_consumption] units of toxins per cycle.<br>"
+	else if(grown_seed.toxin_affinity >= 5 && grown_seed.toxin_affinity <= 7)
+		dat += "It requires [grown_seed.fluid_consumption * 0.5] of both water and toxins per cycle.<br>"
 
-	if(grown_seed.heat_tolerance > 50)
-		dat += "<br>It is well adapted to a range of temperatures."
-	else if(grown_seed.heat_tolerance < 10)
-		dat += "<br>It is very sensitive to temperature shifts."
+	dat += "It thrives in a temperature of [grown_seed.ideal_heat] Kelvin and can tolerate deviations of up to [grown_seed.heat_tolerance] Kelvin.<br>"
 
-	dat += "<br>It thrives in a light level of [grown_seed.ideal_light] lumen[grown_seed.ideal_light == 1 ? "" : "s"]."
+	dat += "It can tolerate pressures between [grown_seed.lowkpa_tolerance] and [grown_seed.highkpa_tolerance] kPa.<br>"
 
-	if(grown_seed.consume_gasses)
-		for(var/gas in grown_seed.consume_gasses)
-			dat += "<br>It will consume [gas] from the environment."
-	if(grown_seed.gas_absorb)
-		dat += "<br>It will turn absorbed gasses into reagents."
-	if(grown_seed.exude_gasses)
-		for(var/gas in grown_seed.exude_gasses)
-			dat += "<br>It will exude [gas] into the environment."
-
-	if(grown_seed.light_tolerance > 7)
-		dat += "<br>It is well adapted to a range of light levels."
-	else if(grown_seed.light_tolerance < 3)
-		dat += "<br>It is very sensitive to light level shifts."
+	dat += "It thrives in a light level of [grown_seed.ideal_light] lumen[grown_seed.ideal_light == 1 ? "" : "s"], and can tolerate a deviation of up to [grown_seed.light_tolerance] lumen[grown_seed.light_tolerance == 1 ? "" : "s"] from it.<br>"
 
+	dat += "It has a toxin affinity of [grown_seed.toxin_affinity] "
 	if(grown_seed.toxin_affinity < 5)
-		dat += "<br>It is highly sensitive to toxins."
+		dat += "and will get damaged if exposed to them.<br>" 
 	else if(grown_seed.toxin_affinity > 7)
-		dat += "<br>It has a remarkable affinity for toxins."
+		dat += "and will require toxins as a fluid, getting damaged if exposed to water.<br>"
+	else if(grown_seed.toxin_affinity >= 5 && grown_seed.toxin_affinity <= 7)
+		dat += "and requires both water and toxins, being able somewhat to tolerate either.<br>"
 
+	dat += "It has a pest tolerance of [grown_seed.pest_tolerance]. "
 	if(grown_seed.pest_tolerance < 30)
-		dat += "<br>It is highly sensitive to pests."
+		dat += "It is highly sensitive to them.<br>"
 	else if(grown_seed.pest_tolerance > 70)
-		dat += "<br>It is remarkably resistant to pests."
+		dat += "It is remarkably resistant to them.<br>"
+	else
+		dat += "It is average.<br>"
```

Comment:
"Its pest resistance is average." would probably make more sense

Reply:
Having it say "Its has a pest resistance of 43. Its pest resistance is average." feels weird. Any ideas?

Reply:
An alternative would be "This is considered average."

Reply:
That's what it says.
"It has a pest resistance of 45. It is average." 
See the pic on the PR.